### ISSUE TITLE
Return sponsored shelf addons in the same order Adzerk returns them

### ISSUE THREAD

**Issue by eviljeff** [mozilla/addons-server#15678]
follow up for #15617 to return the addon in the correct order.

**Comment by ioanarusiczki** [comment]
@eviljeff I see the[ API](https://addons-dev.allizom.org/api/v4/shelves/sponsored/) returns them in a different order after making several requests. 
Request one:
496379 
620031 
620026
620119  
502855 
620004 

Another request:
620026
496379
620004
502855
620119
620031

But that should be tested against Adzerk which I don't know how or if I can access.

**Comment by eviljeff** [comment]
I don't think this is possible to completely test really - Adzerk generates a different order on each request and doesn't keep a log of the previous request orders.  But that they _are_ in different orders for different API requests from AMO is something because the order should have been stable before.

**Comment by ioanarusiczki** [comment]
@eviljeff Then what's happening right now sounds good. 
I'll mark this verified.
